scripts/projectView.js

```diff
@@ -0,0 +1,91 @@
+// Configure a view object, to hold all our functions for dynamic updates and article-related event handlers.
+var portfolioPage = {};
+
+portfolioPage.populateFilter = function(template_id) {
+  var $source = $(template_id).html();
+  var filter = Handlebars.compile($source);
+  return filter({myProjects: projects});
+};
+
+
+Handlebars.registerHelper('list', function(items, options) {
```

Comment:
That is correct. It broke when I moved from 'projects' as an array to Portfolio.all as the array holder.  The 'list' block helper creates the category dropdown elements.